server/api/jurisdictions.py

```diff
@@ -535,6 +536,45 @@ def status(jurisdiction_id: str) -> JurisdictionStatus:
     }
 
 
+@api.route(
+    "/election/<election_id>/jurisdictions/last-login",
+    methods=["GET"],
+)
+@restrict_access([UserType.AUDIT_ADMIN])
+def get_last_login_by_jurisdiction(election: Election):
+    # Look for events after most recent round started, or election creation time
+    # if no rounds exist yet
+    current_round = get_current_round(election)
+    query_timestamp_after = (
+        current_round.created_at if current_round else election.created_at
+    )
+
+    # Query for login activities from users who are related to jurisdictions we found earlier
+    activities = dict(
+        ActivityLogRecord.query.join(
+            User, ActivityLogRecord.info["base"]["user_key"].astext == User.email
+        )
+        .join(User.jurisdictions)
+        .filter(
+            Jurisdiction.election_id == election.id,
+            ActivityLogRecord.organization_id == election.organization_id,
+            ActivityLogRecord.timestamp > query_timestamp_after,
+            ActivityLogRecord.activity_name == "JurisdictionAdminLogin",
+        )
+        .order_by(Jurisdiction.id, ActivityLogRecord.timestamp.desc())
+        .distinct(Jurisdiction.id)
+        .with_entities(Jurisdiction.id, ActivityLogRecord)
+        .all()
+    )
```

Comment:
Will be curious to see how performant this query is in prod - I can't quite recall which indexes exist on the ActivtyLogRecord table

Reply:
Only the primary key index on ID, so this is likely going to be a seq scan